frontend/public/components/nav/nav-header.tsx

```diff
@@ -104,7 +104,7 @@ const NavHeader_: React.FC<NavHeaderProps & StateProps> = ({
 
 const mapStateToProps = (state: RootState): StateProps => ({
   activePerspective: getActivePerspective(state),
-  flags: stateToFlagsObject(state),
+  flags: state[featureReducerName].toObject(),
```

Comment:
Would be still useful to provide a reusable selector here.

Reply:
Will do.